docs/website/versioned_docs/version-3.0.0/command-reference/logs.md

```diff
@@ -0,0 +1,192 @@
+---
+title: odo logs
+---
+
+`odo logs` displays the logs for all the containers odo created for the 
```

Comment:
Thanks for pointing. Thanks a lot for creating a common pattern/template. 👍🏾 
I have made the changes, but not added 'Advanced Usage' since I couldn't think of putting anything in there. PTAL.